climada/util/api_client.py

```diff
@@ -80,15 +79,23 @@ class DataTypeInfo():
     """data_type data from CLIMADA data API."""
     data_type:str
     data_type_group:str
+    status: str
     description:str
-    properties:list = None
+    properties:list
+
+
+@dataclass
+class DataTypeShortInfo():
+    """data_type essentials from CLIMADA data API."""
```

Comment:
This docstring is not clear. What is essentials?

Reply:
👍  replaced by 'name and group"

Reply:
But does it not include more also?

Reply:
obviously not